Packs/GitHub/Integrations/GitHub/GitHub.yml

```diff
@@ -312,6 +312,70 @@ script:
     - contextPath: GitHub.Issue.Organization
       description: The repository organization.
       type: String
+  - arguments:
+    - default: false
+      description: The query line for the search. For more information see the GitHub
+        documentation at https://docs.github.com/en/github/searching-for-information-on-github/searching-code
+      isArray: false
+      name: query
+      required: true
+      secret: false
+    - default: false
+      description: The page number. 
+      isArray: false
+      name: page_number
+      required: false
+      secret: false
+    - default: false
+      description: The size of the requested page. Maximum is 100.
```

Comment:
i think it is better setting the `defaultValue` to `100`

Reply:
I haven't set the defaultvalue because in this specific case user have to pass either page size&page_number or limit.
If I will set the default value, I can't know who passed it, the user or it is the default value.